### ISSUE TITLE
Practical plants scraper readme typo

### ISSUE THREAD

**Issue by 4ydan** [ElektraInitiative/PermaplanT#634]
### Describe the Bug

2. Create .env file from .env.example and fill in the required values:

```shell
cp .env.example .env
```

it should be .env.sample

### Steps to Reproduce the Problem

cp .env.example .env

### Expected Result

to create the env file

### Actual Result

cp: cannot stat '.env.example': No such file or directory

### PermaplanT version

master

### Operating System

Linux

### Browser

-

### Versions of other relevant software/extensions

-

**Comment by 4ydan** [comment]
I think there should be a possibility to start the backend in manual test mode with some existing seeds(and maybe some other test data), without having to go through all these steps.

Somethink like spring profiles in java.

**Comment by Bushuo** [comment]
For a quick fix, you can download an already merged dataset from nextcloud.
There are three for them in the `PermaplanT/Database` folder, but I can't remember which one it is, that worked for me.

**Comment by markus2330** [comment]
A copy is a kind-of-permanent fix as the DB is not updated often. (It is a lot of work to verify the correctness. Most likely we will do the user study with these 3 files as they are.) There are, however, Nextcloud sync tools which would allow you to always sync to the latest files.

The 3 important files are Antagonist.csv  Companions.csv  mergedDatasets.csv (as said correctly by @Bushuo in the folder PermaplanT/Database). I added a README.md, also with the sha256.

**Comment by markus2330** [comment]
I put this to low prior and will look into it when we rescrape the whole DB again (and this won't happen soon).

For our team the provided CSVs are the way to go anyway.

**Comment by 4ydan** [comment]
I downloaded the three files and put them in scraper/data and still get errors after running insert.

![image](https://github.com/ElektraInitiative/PermaplanT/assets/53004139/3f457bc3-7e7a-4be1-b558-fbfaeb3c9971)

**Comment by markus2330** [comment]
> and submitting a form and waiting a couple of days.

@4ydan did you get the key?

**Comment by 4ydan** [comment]
> Do we still need this is? How long is the insertion for you @4ydan?

It took around 15minutes in my wsl with 6 cores. I need to check again when I am home, can also try it with 24 cores this time.

> @4ydan did you get the API key? We should document in Database/README.md who has a key.

I think I did not finish the application process after I realized that I need to wait. I needed an immediate solution and thought the others are faster.

**Comment by 4ydan** [comment]
> Are you sure we don't confuse import vs. completely running the scraper here? Importing should be much faster, for me it takes around 15 seconds.

Yes my bad, I meant inserting. For me coming in fresh and reading the readme from the scraper, it motivated me to fetch everything and mentioned at the end that I can use nextcloud data. I thought this is optional and fetching is the way to go.
